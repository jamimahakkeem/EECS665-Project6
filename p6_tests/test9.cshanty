record TheRecord{
	int a;
	int b;
	int c;
}

void f(){
	TheRecord inst;
	inst[a] = 1;
}