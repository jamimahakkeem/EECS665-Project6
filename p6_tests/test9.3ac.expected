[BEGIN GLOBALS]
[END GLOBALS]
[BEGIN f LOCALS]
inst (local var of 24 bytes)
[addrTmp0] (tmp loc of 8 bytes)
[END f LOCALS]
fun_f:      enter f
            [addrTmp0] := inst ADD64 16
            [[addrTmp0]] := 1
lbl_0:      leave f
